tests/transports/test_base.py

```diff
@@ -120,7 +120,7 @@ def test_flush_time(mock_send, caplog, elasticapm_client):
         transport = Transport(client=elasticapm_client)
         transport.start_thread()
         # let first run finish
-        time.sleep(0.2)
+        time.sleep(0.3)
```

Comment:
could we avoid these changes by setting `cloud_provider` to `False` for the test client ([here](https://github.com/elastic/apm-agent-python/blob/master/tests/fixtures.py#L150))?

Reply:
Actually, yes. Good idea. I'll make that change later today.